### PR TITLE
Resolve comments on #36

### PR DESCRIPTION
**Pull request by shivampotdar**
The alignment in rslt_mux for div was redundant, fixed that.
In case of ALTOPS, the +3 part is correct and verified for all 3 cases (1/4/6 stage). It is basically second issuing the multiplication instruction deliberately, even though the actual latency is smaller, we flop the result upto M4_MUL_LATENCY.

### CODE REVIEW

**Comment by shivampotdar** [comment]
@stevehoover @ahadnagy thoughts about formal runs for all 3 cases of pipelining? Just to see in case we made any hardcoded change which does not work for all.

**Comment by ahadnagy** [comment]
> @stevehoover @ahadnagy thoughts about formal runs for all 3 cases of pipelining? Just to see in case we made any hardcoded change which does not work for all.

I would definitely recommend running the procedure for all the configurations, at least locally.

**Comment by shivampotdar** [comment]
> > @stevehoover @ahadnagy thoughts about formal runs for all 3 cases of pipelining? Just to see in case we made any hardcoded change which does not work for all.
> 
> I would definitely recommend running the procedure for all the configurations, at least locally.

https://travis-ci.com/github/shivampotdar/warp-v/jobs/357847448
https://github.com/shivampotdar/warp-v/blob/m_ext_formal/.travis.yml

1 and 4 stage up and running. Travis runs them as parallel jobs, so effective build time is still same
For 6 stage I need some guidance on how to set the values in `checks.cfg`. We probably need to review them for the 4 stage case too.

File: warp-v.tlv [hunk 1]
```diff
@@ -125,8 +125,8 @@ m4+definitions(['
    //     destination register, raw value, rs1/rs2/rd) depending on where they are consumed
    //     need to be retained. $ANY construct is used to make this logic generic and use-dependent. 
    //   o In case of loads, the /orig_load_inst scope is used to hook up the 
-   //     CPU pipeline to the |mem pipeline in first pipestage (since loads are speculative),
-   //     and other signals (such as load value and mask) are pushed in later in the pipeline.
+   //     CPU pipeline to the |mem pipeline in first pipestage to reserve slot for the load
```
Comment (shivampotdar) [inline warp-v.tlv:128]: mem -> cpu

**Comment by shivampotdar** [comment]
@stevehoover ready to merge!

**Pull request merged**